apps/src/lab2/views/components/TextToSpeech.tsx

```diff
@@ -80,6 +89,8 @@ const TextToSpeech: React.FunctionComponent<TextToSpeechProps> = ({text}) => {
         isPlaying && moduleStyles.playButtonPlaying
       )}
       onClick={playText}
+      onKeyDown={handleKeyDown}
```

Comment:
Out of curiosity, does this not happen natively (using the `onClick` handler as the `onKeyDown` handler)?

Reply:
It wasn't working natively- I suspect there was some other event listener that was interfering. I say this because I had to preventDefault to keep this new handler from calling the onClick twice. That being said- if you can figure out why it wasn't working in the first place I'd be happy to rewrite this!

Reply:
Fascinating... i just figured out that the enter key is working natively but only if I *do NOT have a screenreader turned on. If I turn on my screenreader, the enter key stops working.

Reply:
Hmm interesting - I would be curious if this is the case across all buttons on our site or just this one? One thing that might be interfering is Music Lab's [KeyHandler](https://github.com/code-dot-org/code-dot-org/blob/2450d1674d40c7cda98274f20c927a6b636bafa3/apps/src/music/views/KeyHandler.tsx#L15), but I don't think we have a mapping for the Enter button...

Reply:
It's an issue across many of our buttons yes, but we've instrumented an onKeyDown in several places to combat this!